packages/core/src/shared/ui/pickerPrompter.ts

```diff
@@ -143,6 +144,22 @@ export function createQuickPick<T>(
     assign(mergedOptions, picker)
     picker.buttons = mergedOptions.buttons ?? []
 
+    picker.onDidTriggerItemButton(async (event) => {
+        const metadataManager = MetadataManager.getInstance()
+        if (event.button.tooltip !== 'Open in Serverless Land') {
+            return
+        }
+        const selectedPattern = event.item
+        if (!selectedPattern) {
+            return
+        }
+        const patternUrl = metadataManager.getUrl(selectedPattern.label)
+        if (!patternUrl) {
+            return
+        }
+        await vscode.env.openExternal(vscode.Uri.parse(patternUrl))
```

Comment:
use `openUrl` (unless the url has PII) https://github.com/aws/aws-toolkit-vscode/blob/62d3ec19969c5eef6ee22f7ed155a0c32452ebcf/packages/core/src/shared/utilities/vsCodeUtils.ts#L217